pkg/grpc/api.go

```diff
@@ -1337,28 +1356,24 @@ func (a *api) SubscribeConfiguration(request *runtimev1pb.SubscribeConfiguration
 		apiServerLogger.Debug(err)
 		return err
 	}
-	sort.Slice(request.Keys, func(i, j int) bool {
-		return request.Keys[i] < request.Keys[j]
-	})
 
-	subscribeKeys := make([]string, 0)
+	sort.Strings(request.Keys)
 
 	// TODO(@halspang) provide a switch to use just resiliency or this.
 
-	if len(request.Keys) > 0 {
-		subscribeKeys = append(subscribeKeys, request.Keys...)
-	}
```

Comment:
Why are we no longer removing empty keys? This seems important.

Reply:
Why is this block of code removed?

Reply:
I am putting request.keys below already, this was redundant.